build.gradle

```diff
@@ -18,20 +18,23 @@ plugins {
     id 'io.gitlab.arturbosch.detekt' version '1.0.0.RC6-4'
     id 'com.diffplug.gradle.spotless' version '3.10.0'
     id 'net.ltgt.errorprone' version '0.0.13'
+    id "com.adarshr.test-logger" version "1.2.0"
 }
 
 description = 'SAML Conformance Test Kit'
 
 project.ext {
     project_version = '1.0-SNAPSHOT'
-    kotlin_test_version = '2.0.7'
+    kotlin_test_version = '3.1.0-RC2'
     rest_assured_version = '2.9.0'
     org_slf4j_version = '1.7.1'
-    junit_version = '4.12'
     guava_version = '24.0-jre'
     detekt_version = '1.0.0.RC6-4'
     java_encoding = 'UTF-8'
     static_log_version = '2.2.0'
+    mockk_version = '1.7.15'
```

Comment:
✏️ It doesn't look like this is being used anywhere

Reply:
It will likely be in use when I write my next test. If not, I'll remove it then.